### PR TITLE
[NO-TICKET] Theme color organization

### PR DESCRIPTION
**Pull request by line47**
## Summary

- Better organized and categorized the theme color page

## How to test

1. `yarn start` and visit the theme color page and see the colors contain a neutrals category as well as sub categories for status (success, warn, error) 

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title`

### If this is a change to code:

- [ ] Created or updated unit tests to cover the logic added
- [ ] If necessary, updated unit-test snapshots (`yarn test:unit:update`) and browser-test snapshots (`yarn test:browser:update`)

### If this is a change to documentation:

- [ ] Checked for spelling and grammatical errors

### CODE REVIEW

File: packages/docs/content/foundation/theme-colors.mdx [hunk 1]
```diff
@@ -60,11 +60,6 @@ The guidelines indicate how colors should be applied, noting where some should b
     'primary',
     'primary-darker',
     'primary-darkest',
-    'black',
```
Comment (line47) [inline packages/docs/content/foundation/theme-colors.mdx:63]: Moved these to neutrals section to make space for additional primary shades

File: packages/docs/content/foundation/theme-colors.mdx [hunk 2]
```diff
@@ -79,13 +74,29 @@ These are accent colors to provide additional lightness and style to pages looki
 
 <ColorSwatchList
   colorNames={[
+    'secondary-lightest',
```
Comment (line47) [inline packages/docs/content/foundation/theme-colors.mdx:77]: Ordered from lightest to darkest like the system color page

Comment (line47) [inline packages/docs/content/foundation/theme-colors.mdx:77]: Ordered from lightest to darkest like the system color page

File: packages/docs/content/foundation/theme-colors.mdx [hunk 3]
```diff
@@ -79,13 +74,29 @@ These are accent colors to provide additional lightness and style to pages looki
...
     'secondary-darker',
     'secondary-darkest',
-    'secondary-light',
-    'secondary-lighter',
-    'secondary-lightest',
+  ]}
+  preface="--color-"
+/>
+
+## Neutral colors
```
Comment (line47) [inline packages/docs/content/foundation/theme-colors.mdx:88]: Added section for neutrals

Comment (line47) [inline packages/docs/content/foundation/theme-colors.mdx:88]: Added section for neutrals

File: packages/docs/content/foundation/theme-colors.mdx [hunk 4]
```diff
@@ -101,12 +112,7 @@ These colors are used largely for background blocks and large content areas. Whe
 <ColorSwatchList
   colorNames={[
     'background',
-    'background-inverse',
```
Comment (line47) [inline packages/docs/content/foundation/theme-colors.mdx:104]: Removed gray colors from this to make space for more intentional color naming in the future

File: packages/docs/content/foundation/theme-colors.mdx [hunk 5]
```diff
@@ -118,33 +124,47 @@ These colors are used largely for background blocks and large content areas. Whe
 These colors are used largely to indicate the status of something.
 Be cautious using these colors to signify something that may contradict what most people would expect the colors to be used for.
 
-Status colors can also be used with these aliases where `green`, `gold`, and `red` are identical to `success`, `warn`, and `error`. Use these alternate naming options when statuses aren't approproiate or give enough detail. For example, in some situations, `--color-red-dark` might be more accurate and descriptive than `--color-error-darker`.
```
Comment (line47) [inline packages/docs/content/foundation/theme-colors.mdx:121]: Removed b/c we are going to be taking non-semantica colors out of the theme in future work and don't want to promote the use of `--color-red-dark` and simialr

File: packages/docs/content/foundation/theme-colors.mdx [hunk 6]
```diff
@@ -118,33 +124,47 @@ These colors are used largely for background blocks and large content areas. Whe
...
-
 </ThemeContent>
 
+### Success
+
 <ColorSwatchList
   colorNames={[
-    'success-darkest',
```
Comment (line47) [inline packages/docs/content/foundation/theme-colors.mdx:127]: Break statues down into success, warn and error instead of one big list of colors

**Review by brentschneider — APPROVED** [review]
LGTM too 👍🏼

**Comment by line47** [comment]
Trying to catch up with main to get things merged :joy:

**Pull request merged**